internal/handlers/auth_handlers.go

```
@@ -0,0 +1,254 @@
+		return
+	}
+
+	// Generate new tokens
+	accessToken, refreshToken, err := h.authMiddleware.GenerateTokens(user)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to generate tokens"})
+		return
+	}
+
+	// Revoke old refresh token
+	h.authMiddleware.RevokeRefreshToken(req.RefreshToken, userID.(uint))
```

golangci-lint (1.64.8)

Error return value of `h.authMiddleware.RevokeRefreshToken` is not checked

(errcheck)